packages/flutter/lib/src/material/button.dart

```diff
@@ -10,8 +10,28 @@ import 'ink_well.dart';
 import 'material.dart';
 import 'theme.dart';
 
-enum ButtonColor { normal, accent }
+/// Whether a button should use the accent color for its text.
+///
+/// See also:
+///
+///  * [ButtonTheme]
+///  * [RaisedButton]
+///  * [FlatButton]
+enum ButtonColor {
+  /// The button should use the normal color (e.g., black or white depending on the [ThemeData.brightness]) for its text.
```

Comment:
That seems sensible, but we don't do that currently.  Are there examples of icons in raised or flat buttons?